requirements.txt

```diff
@@ -59,6 +59,7 @@ pytest-asyncio==0.20.3
 pytest-cov==4.1.0
 pytest-timeout==2.1.0
 pytest-xdist==3.3.1
+requests
```

Comment:
You are requesting that all pymodbus users add your requirement, that seems wrong.

I suggest you write it in the example as a demand, and check if it is installed with importlib.metadata.

Reply:
For what it's worth, I am pretty sure I added this dependency in the "development" section of `requirements.txt`, so it's not required for the actual `pymodbus` package.  That being said, I can make it dynamic with `importlib.metadata` as requested

Reply:
Okay, moved to `contextlib.suppress` for `requests` not being installed, and removed it from `requirements.txt`